pinot-plugins/pinot-batch-ingestion/pinot-batch-ingestion-spark/src/main/java/org/apache/pinot/plugin/ingestion/batch/spark/SparkSegmentMetadataPushJobRunner.java

```diff
@@ -111,6 +112,7 @@ public void run() {
         @Override
         public void call(String segmentTarPath)
             throws Exception {
+          PluginManager.get().init();
```

Comment:
Should we init the plugin manager and also register PinotFS  in `init()`? Seems the PinotFS is used in `run()`. Same for other runners

Reply:
+1 on that. Since we have init() method, can we just initialize the plugin manager there?

Reply:
this is inside the sparkContext, which will be run on each spark worker, the init code is only running on the driver

Reply:
In that case, do we need to also init the plugin manager for the driver?

Reply:
that is from the outside LauncherJob code.